src/components/checkbox-drawer/checkbox-drawer.js

```diff
@@ -92,20 +91,22 @@ class CheckboxDrawer extends LocalizeLabsElement(SkeletonMixin(LitElement)) {
 	}
 
 	async _onExpandCollapseContentToggled(e) {
-		const action = e.target.expanded ? 'expand' : 'collapse';
+		const action = e.target.expanded ? 'expandComplete' : 'collapseComplete';
+		const eventName = e.target.expanded ? 'd2l-labs-checkbox-drawer-expand' : 'd2l-labs-checkbox-drawer-collapse';
 
 		this.dispatchEvent(
-			new CustomEvent(`d2l-checkbox-drawer-${action}`, {
+			new CustomEvent(eventName, {
 				bubbles: true,
 				composed: false,
 				detail: e.detail
 			})
 		);
 
-		const content = this.shadowRoot?.querySelector('d2l-expand-collapse-content');
-		content.ariaBusy = 'true';
-		await e.detail[`${action}Complete`];
-		content.ariaBusy = 'false';
+		this._expandCollapseBusy = true;
+		await this.updateComplete;
```

Comment:
You shouldn't need this `await this.updateComplete` since you don't really care about waiting for the next render before awaiting the detail.